src/main/java/core/basesyntax/SupplyOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class SupplyOperation implements OperationHandler {
```

Comment:
The `SupplyOperation` class should be placed in the `strategy` package to follow the checklist requirement of organizing classes by their purpose.